data/_redirects

```diff
@@ -1,6 +1,7 @@
 /blog/author  /blog
 /people/*  /people#:splat
-/case-studies/*  /project-stories
+/case-studies/*  /product-stories
+/project-stories/* /product-stories
```

Comment:
do we want `/:splat` on the other end so that someone visiting `/project-stories/bucks-cc` ends up at `/product-stories/bucks-cc` instead of `/product-stories` ?